WordPress/Classes/ViewRelated/Reader/Sorting/ReaderSortingOptionButton.swift

```diff
@@ -0,0 +1,124 @@
+import UIKit
+
+extension ReaderSortingOption {
+
+    var localizedDescription: String? {
+        // TODO: check if strings are localized
+        switch self {
+        case .date:
+            return NSLocalizedString("Recent", comment: "Sorting option description")
+        case .popularity:
+            return NSLocalizedString("Popular", comment: "Sorting option description")
```

Comment:
can we use comments that clarify a little better what the strings actually are? Like "Description of the popularity sorting option in the Discover tab"